app/uk/gov/hmrc/securemessage/controllers/models/generic/ConversationMetadata.scala

```diff
@@ -46,22 +48,16 @@ object ConversationMetaData {
     )
   }
 
-  private def findLatestMessage(coreConversation: Conversation): Option[Message] =
-    coreConversation.messages.sortWith(_.created.getMillis > _.created.getMillis).headOption match {
-      case Some(message) => Some(message)
-      case _             => None
-    }
+  private def findLatestMessage(coreConversation: Conversation): Message =
+    coreConversation.messages.sortBy(_.created.getMillis).reverse.head
```

Comment:
can we use headOption ? we may atleast have a message for a conversation , but still I feel its better have headOption

Reply:
May be its safe becuase its nonempty list ?